products/jenkins.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Jenkins
+permalink: /jenkins
+iconSlug: jenkins
+category: server-app
+releasePolicyLink: https://www.jenkins.io/download/
+changelogTemplate: https://github.com/jenkinsci/jenkins/releases/tag/jenkins-__LATEST__
+activeSupportColumn: true
+releaseDateColumn: true
+auto:
+-   git: https://github.com/jenkinsci/jenkins.git
+releases:
+-   releaseCycle: "2"
```

Comment:
maybe a releaseLabel: "2.X branch" might be nice

Reply:
For the time being I do not know. Ideally I would like that both the weekly and LTS release lines be updated automatically. But I do not know if it is possible (waiting for @captn3m0 on this).

Reply:
From my understanding, the latest `2.x` release will always be the weekly release. We could just set `releaseLabel: Stable` here instead as well (The current 2 also works fine).

Reply:
Just set `releaseLabel` to "Regular" while keeping `releaseCycle: "2"`. At least this way `latest.py` don't complain with "not included" versions. Let's try with this and see how it behaves.